tests/simulator/test_initialization.py

```diff
@@ -328,13 +326,12 @@ def _test_B_is_as_provided_by_user(self, dim, interp_order, **kwargs):
 
     def _test_bulkvel_is_as_provided_by_user(self, dim, interp_order):
         hier = self.getHierarchy(
+            dim,
             interp_order,
             {"L0": {"B0": nDBox(dim, 10, 19)}},
             "moments",
-            nbr_part_per_cell=100,
+            nbr_part_per_cell=100,  # ppc needs to be 10000?
```

Comment:
question?

Reply:
moved, not added, but looks redundant too